shared/src/vyxal/Elements.scala

```diff
@@ -4049,6 +4049,31 @@ object Elements:
       false,
       "a: lst => indices of truthy elements of a",
     ) { a => ListHelpers.truthyIndices(ListHelpers.makeIterable(a)) },
+    addPart(
+      Monad,
+      "øA",
+      "Letter Number Swap",
+      List(
+        "letter-to-number",
+        "number-to-letter",
+        "letter-number-swap",
+        "number-letter-swap",
+        "a1-swap",
+      ),
+      true,
+      "a: str -> the index of a in the alphabet (one-indexed)",
+      "a: int -> the a-th letter of the alphabet (one-indexed)",
+    ) {
+      case a: VNum =>
+        "abcdefghijklmnopqrstuvwxyz" (math.floorMod((a - 1).toInt, 26)).toString
```

Comment:
I'd prefer converting to an `Int` at the end, in case the number is too big to fit inside an `Int`. Maybe something like this?
```suggestion
        "abcdefghijklmnopqrstuvwxyz" (((a - 1).floor % 26).toInt).toString
```
Also, `<string literal> (<arg>).<method>` is kinda hard to read. I'd suggest using `.charAt` because that feels clearer than `.apply`, or better, extract `"abc...xyz"` to a `val` inside `StringHelpers`